documentation/modules/configuring/proc-scaling-down-node-pools.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-scaling-down-node-pools-{context}']
+= (Preview) Removing nodes from a node pool
+
+[role="_abstract"]
+This procedure describes how to scale down a node pool to remove nodes.
+
+The steps assume that you are using ZooKeeper for cluster management. 
+The operation is not possible for clusters using KRaft mode. 
```

Comment:
I think we can remove this.

Reply:
Removed for this. Does that mean we retain for scaling up and moving nodes in node pools?

Reply:
I think with regard to broker nodes, everything should work fine. The controller nodes not really, but that is a different story - I would not include it into these procedures.

Reply:
Thanks. I've removed from the other procedures